libsplinter/src/service/lifecycle.rs

```diff
@@ -12,30 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! Contains `Lifecycle` trait.
+
 use crate::error::InternalError;
 use crate::store::command::StoreCommand;
 
 use super::{ArgumentsConverter, FullyQualifiedServiceId};
 
+/// Every service type needs to implement this trait. It will be used by the "LifecycleExecutor"
```

Comment:
This should be a one-line summary for the trait - this line is more of a description what/why it is required. Likewise in many other doc comments for traits.

Reply:
fixed